$MirOS: ports/www/firesomething/patches/patch-nsprpub_pr_src_misc_prdtoa_c,v 1.3 2009/08/04 08:26:18 tg Exp $
$OpenBSD: patch-nsprpub_pr_src_misc_prdtoa_c,v 1.1 2005/11/03 00:59:43 pvalchev Exp $
--- nsprpub/pr/src/misc/prdtoa.c.orig	Tue Oct  6 01:05:01 2009
+++ nsprpub/pr/src/misc/prdtoa.c	Sun Nov 15 00:10:00 2009
@@ -854,7 +854,7 @@ mult
 	wc = wa + wb;
 	if (wc > a->maxwds)
 		k++;
-	c = Balloc(k);
+	c = Balloc(k+2);
 	for(x = c->x, xa = x + wc; x < xa; x++)
 		*x = 0;
 	xa = a->x;
